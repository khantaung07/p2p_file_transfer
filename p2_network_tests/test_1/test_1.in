CONNECT
CONNECT 1.1.1.1.1:777
CONNECT 127.0.0.1:9001
PEERS
DISCONNECT
DISCONNECT 127.0.0.1:9999
DISCONNECT 127.0.0.1:9001
PEERS
QUIT
