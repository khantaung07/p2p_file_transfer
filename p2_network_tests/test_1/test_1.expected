Missing address and port argument
Unable to connect to request peer
Connection established with peer
Connected to:

1. 127.0.0.1:9001
Missing address and port argument
Unknown peer, not connected
Disconnected from peer
Not connected to any peers
